content/en/account_management/rbac/_index.md

```diff
@@ -1,50 +1,144 @@
 </div>
 
-## Roles
+Manage your custom roles through the Datadog application, the [Datadog Role API][5], or SAML directly. Find below how to create, update, delete a role. See the [Datadog Role permissions][6] documentation for more information about available permissions.
 
-Roles categorize users and define what account permissions those users have, such as what data they can read or what account assets they can modify. By default, Datadog offers three roles, and you can create custom roles so you can define a better mapping between your users and their permissions.
+### Create a custom role
 
-By granting permissions to roles, any user who is associated with that role receives that permission. When users are associated with multiple roles, they receive all the permissions granted to each of their roles. The more roles a user is associated with, the more access they have within a Datadog account.
+You can create custom roles with:
 
-## Out of the Box Roles
+{{< tabs >}}
+{{% tab "Datadog application" %}}
 
-The following roles are available out of the box. By default, users are associated with one of these three roles:
+To create a custom role:
 
-| Role                   | Description                                                                                 |
-|------------------------|---------------------------------------------------------------------------------------------|
-| Datadog Admin          | Has read and write permissions on all data and assets in a Datadog account.                 |
-| Datadog Standard Role  | Has read permissions on all data and write permissions on most assets in a Datadog account. |
-| Datadog Read Only Role | Has read permissions on all data and write permissions on no assets in a Datadog account.   |
+1. Go to your [Datadog Roles Page][1].
+2. Select **New Role** in the upper right corner of the page.
+3. Give a name to your role.
+4. Optional - Assing a set of permissions to your Role. See the [Datadog Role permissions][2] documentation for more information about available permissions.
 
-## Custom Roles
+{{< img src="account_management/rbac/create_role.png" alt="Create a custom Role"  style="width:90%;">}}
+
+Once a role is created you can [add this role to existing users][3].
+
+
+[1]: https://app.datadoghq.com/access/roles
+[2]: /account_management/rbac/permissions
+[3]: /account_management/users/#edit-a-user-roles
+{{% /tab %}}
+{{% tab "API" %}}
+
+Find example to create a Role in the [Datadog Create Role API documentation][1].
+
+[1]: /api/#create-role
+{{% /tab %}}
+{{% tab "SAML" %}}
 
 Custom roles are imported via SAML integrations from identity providers. Datadog receives the user groups from your IdP and automatically generates roles to match those user groups. In this case, users who sign in via the IdP are automatically associated with those roles and have the permissions that are granted to those roles.
```

Comment:
This functionality is being deprecated. Roles must be explicitly created either via the API or the Application going forward.

Reply:
Removed that section